assets/base.css

```diff
@@ -2231,6 +2231,23 @@ details[open] > .header__icon--menu .icon-hamburger {
     transform var(--duration-default) ease;
 }
 
+.header__submenu.list-menu--disclosure {
+  --shadow-horizontal-offset: var(--popup-drawer-shadow-horizontal-offset);
+  --shadow-vertical-offset: var(--popup-drawer-shadow-vertical-offset);
+  --shadow-blur-radius: var(--popup-drawer-shadow-blur-radius);
+  --shadow-opacity: var(--popup-drawer-shadow-opacity);
+  border-radius: var(--popup-drawer-corner-radius);
+  border-color: rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
+  border-style: solid;
+  border-width: 0 0 var(--popup-drawer-border-width);
```

Comment:
We're only applying a border to the bottom?
![](https://screenshot.click/21-11-2siil-x5gie.png)